src/App.js

```diff
@@ -25,8 +40,18 @@ export class App extends Component {
           <div className="o-container">
             <div className="c-site-content">
               <Switch>
+                <Route
+                  path="/registrationSite"
+                  exact
+                  component={RegistrationSite}
+                />
+                {this.state.user ? (
+                  <Route path="/" exact component={Catalog} />
+                ) : (
+                  <Route path="/" exact component={LoginSite} />
+                )}
+
                 <Route path="/add-form" component={AddForm} />
-                <Route path="/" exact component={Catalog} />
                 <Route path="/calendar" component={Calendar} />
                 <Route path="/todolist" component={Todolist} />
                 <Route path="/wishlist" component={Wishlist} />
```

Comment:
Those should be put inside the `this.state.user ? .... : ....` expression above as well